File: java/Tarjeta.java
import java.sql.Date;

class Tarjeta extends Payment {
    Integer id;
    Integer cardNumber;
    Integer CVV;
    Date dueDate;
    public Tarjeta(Account name, String mail, String password, Integer id,
    Integer cardNumber,Integer CVV,Date dueDate){
        super(name, mail, password);
        this.id = id;
        this.cardNumber = cardNumber;
        this.CVV = CVV;
        this.dueDate = dueDate
    }
}

File: python/tarjeta.py
from account import Account

class Tarjeta(Account):
    cardNumber  = int
    cvv         = int
    dueDate     = int

    def __init__(self, name, mail, password,cardNumber, cvv, dueDate):
        super().__init__(name, mail, password)
        self.cardNumber = cardNumber
        self.cvv = cvv
        self.dueDate = dueDate


